nengo_gui/components/pointer.py

```diff
@@ -43,16 +45,18 @@ def remove_nengo_objects(self, page):
 
     def gather_data(self, t, x):
         vocab = self.vocab_out
-        m = np.dot(vocab.vectors, x)
-        matches = [(mm, vocab.keys[i]) for i, mm in enumerate(m) if mm > 0.01]
+        key_similarity = np.dot(vocab.vectors, x)
```

Comment:
I would name this `key_similarities` because it is all the similarities for all of the vectors (if I'm not mistaken).